src/Redux/books/books.js

```diff
@@ -0,0 +1,59 @@
+const ADD_BOOK = 'books/ADD_BOOK';
+const REMOVE_BOOK = 'books/REMOVE_BOOK';
+// const GET_BOOKS = 'books/GET_BOOKS';
+
+const initialState = [
+  {
+    id: 1,
+    author: 'Tamara',
+    title: 'girl fusion',
+    category: '',
+  },
+  {
+    id: 2,
+    author: 'Tamara',
+    title: '  The church girl',
+    category: '',
+  },
+  {
+    id: 3,
+    author: 'Tamara,Emmanuel',
+    title: 'The girl i Love',
+    category: '',
+  },
+];
+
+// Actions
+
+export const addBook = (book) => ({ type: ADD_BOOK, book });
+
+// export const getBook = (payload) => ({ type: ADD_BOOK, payload });
+
+export const deleteBook = (id) => (
+  {
+    type: REMOVE_BOOK, id,
+  });
+
+// Reducers
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK: {
+      const bookObj = {
+        id: state.length + 1,
+        author: action.book.author,
+        title: action.book.title,
+      };
+      return [...state, bookObj];
+    }
+
+    // case GET_BOOKS:
+    //   return [...state, action.payload];
```

Comment:
- [ ] Kindly remove all unnecessary commented-out code in this file. Removing such codes makes your code clean and it shows that you adopt best JavaScript practices when building an application. 👍